### PR TITLE
Removing the shadow in the icon

### PR DESCRIPTION
**Pull request by altoidkind**

### CODE REVIEW

**Comment by escapedcat** [comment]
Thanks!  
Please describe what your PR is solving? Can existing issue be referenced?  
Maybe adding a screenshot helps?  
Would be great to understand you PR better to get this moving faster.

File: src/app/screens/Receive.tsx [hunk 1]
```diff
@@ -232,16 +232,6 @@ function Receive() {
                   }
                 />
               </div>
-              <a
```
Comment (escapedcat) [inline src/app/screens/Receive.tsx:235]: How is this related to removing a shadow?

Comment (bumi) [reply src/app/screens/Receive.tsx:235]: I think that's a left over from: https://github.com/getAlby/lightning-browser-extension/pull/657
